vtex-apps/my-security
# Chore/configure project

#### What problem is this solving?

Configure the project from its creation template

#### How to test it?

Go into
https://rafaprtest6--storecomponents.myvtex.com/login?returnUrl=%2Faccount%23%2Fsecurity
then log in (twice) and you should see the extension `<Greeting />` component inside `my-account`:

![image](https://user-images.githubusercontent.com/22064061/98593926-a6a0a100-22b2-11eb-8fea-3168ed041df7.png)

#### How does this PR make you feel? [:link:](http://gph.is/VwQaUY)

![temp](https://user-images.githubusercontent.com/22064061/98594449-6f7ebf80-22b3-11eb-8809-3594bf70fe42.gif)
